projects/azure/base-images/ci-base.yaml

```diff
@@ -78,6 +78,10 @@ items:
         RUN curl -sL https://github.com/openshift/origin/releases/download/v3.11.0/openshift-origin-client-tools-v3.11.0-0cbc58b-linux-64bit.tar.gz | tar -C /tmp -xz
         RUN mv /tmp/openshift-origin-client-tools-v3.11.0-0cbc58b-linux-64bit/oc /usr/local/bin
 
+        # install kubectl for tests
+        RUN curl -s -L -o /usr/local/bin/kubectl https://storage.googleapis.com/kubernetes-release/release/$(curl -s https://storage.googleapis.com/kubernetes-release/release/stable.txt)/bin/linux/amd64/kubectl
```

Comment:
This will get us the latest `kubectl`, which is currently v1.15.0, but my understanding is that the `kubectl` client is only guaranteed to be compatible with clusters one major version different. Do we want to pin this to v1.12 for OpenShift 3.11?

Reply:
you already have kubectl from the tgz referenced on line 78, copy it from there